bin/build-metadata.js

```diff
@@ -214,6 +214,18 @@ function removeAllNumberKeys( obj ) {
 	return _.omitBy( obj, function( val, key ) { return /^\d+$/.test( key ); } );
 }
 
+function removeRegionCodeAndCountryDialCodeIfSameWithCountryDialCode( countryData ) {
+	for ( var key in countryData ) {
+		if ( countryData.hasOwnProperty( key ) ) {
+			if ( ! countryData[ key ].regionCode && ( countryData[ key ].countryDialCode + ( countryData[ key ].regionCode || '' ) ) === countryData[ key ].dialCode ) {
```

Comment:
Good point, forgot we need to download all that data (though it would probably be neatly compressed).